backend/api/Controllers/RobotController.cs

```diff
@@ -173,6 +175,97 @@ public async Task<ActionResult<RobotResponse>> CreateRobot([FromBody] CreateRobo
             }
         }
 
+        /// <summary>
```

Comment:
why is this part of the PR? Just wondering if it's intentional or something in the rebase because i thought we already had this functionality but maybe not in the robot controller